iceoryx_posh/source/runtime/posh_runtime.cpp

```diff
@@ -517,6 +521,43 @@ PoshRuntime::requestConditionVariableFromRoudi(const MqMessage& sendBuffer) noex
     return cxx::error<MqMessageErrorType>(MqMessageErrorType::REQUEST_CONDITION_VARIABLE_WRONG_MESSAGE_QUEUE_RESPONSE);
 }
 
+cxx::expected<popo::EventVariableData*, MqMessageErrorType>
+PoshRuntime::requestEventVariableFromRoudi(const MqMessage& sendBuffer) noexcept
+{
+    MqMessage receiveBuffer;
+    if (sendRequestToRouDi(sendBuffer, receiveBuffer) && (3U == receiveBuffer.getNumberOfElements()))
```

Comment:
oh boy, we definitely have to create a sane way to check for an error response :D

Reply:
You are right ... a task for our future selfs